Sources/MMIO/BitFieldProjectable.swift

```diff
@@ -43,6 +43,16 @@ extension Bool: BitFieldProjectable {
   }
 }
 
+extension RawRepresentable where Self: BitFieldProjectable, RawValue: FixedWidthInteger {
+  public init<Storage>(storage: Storage) where Storage: FixedWidthInteger & UnsignedInteger {
+      self.init(rawValue: RawValue(storage))!
```

Comment:
Definitely need to use a precondition failure with a message explaining the issue